README.md

```diff
@@ -41,74 +40,89 @@ brew install cmake git wget curl bash-completion qemu
 python3 -m pip install --user -U  colcon-common-extensions rosdep vcstool
 ```
 
-[Install Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/)
+[Install Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/) on Mac.
+
+
+### Installing from source
+
+#### Latest (unstable development - `master` branch)
+Please follow those instructions if you plan to contribute to this repository.
+
+* Install all software dependencies required for ROS 2 development by following the [ROS 2 documentation](https://index.ros.org/doc/ros2/Installation/Latest-Development-Setup/)
+* Checkout the source code and compile it as follow
+
+```bash
+cd my_ros2_ws/src
+git clone https://github.com/ros-tooling/cross_compile
+
+# Install dependencies required by the cross-compile tool
+cd cross_compile
+pip3 install -r requirements.txt
```

Comment:
This is not needed since:
a) We already ask to install ROS2 
b) requirements are part of the `setup.py` file which is taken care of when you run `colcon build`.